tests/backup/backup_helper.go

```diff
@@ -1490,3 +1493,54 @@ func IsPresent(dataSlice interface{}, data interface{}) bool {
 	return false
 
 }
+
+// return current stork image.
+func getStorkImage() (string, error) {
+	storkDeployment, err := apps.Instance().GetDeployment(storkDeploymentName, storkDeploymentNamespace)
+	if err != nil {
+		return "", err
+	}
+	storkImage := storkDeployment.Spec.Template.Spec.Containers[0].Image
+	return storkImage, nil
+}
+
+// Upgrade the stork to required version.
+func upgradeStorkVersion(reqStorkImage string) (v1.ConditionStatus, error) {
```

Comment:
change the variable name from `reqStorkImage` to `storkVersionToUpgrade`

Reply:
Why are we returning `v1.ConditionStatus` rather than a simple bool?

Reply:
change the variable to storkVersionToUpgrade.
Add code to handle v1.ConditionStatus ad bool